kumascript/macros/WebkitBug.ejs

```diff
@@ -1,4 +1,8 @@
 <%
+// Condition for removal: no more use in any content (Feb 2023: 18 occurrences)
+// 4 occurrences left in en-US
+mdn.deprecated("This macro is deprecated; replace with `[WebKit bug <bug_id>](https://webkit.org/b/<bug_id>)`.");
```

Comment:
The `macro-usage-report` determines if a macro is deprecated by looking for `mdn.deprecated()`, so it won't understand this macro is deprecated:

https://github.com/mdn/yari/blob/945e7f5898c4222a0d7abdce58dc0aceec68ef02/tool/macro-usage-report.ts#L120-L125

The reason for this is that there are macros that deprecated specific parameters, and this is where the message parameter has been exclusively used for so far:

https://github.com/mdn/yari/blob/945e7f5898c4222a0d7abdce58dc0aceec68ef02/kumascript/macros/Compat.ejs#L33-L36

Maybe the easiest way to resolve this is to introduce a method `mdn.deprecatedParameter()` for this last case, and looking for `mdn.deprecated(` in the `macro-usage-report`.

Does this make sense to you? And if so, could you open a separate PR for this?